hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/QueryEndpoint.java

```diff
@@ -2,10 +2,7 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import pl.allegro.tech.hermes.api.Group;
-import pl.allegro.tech.hermes.api.Subscription;
-import pl.allegro.tech.hermes.api.Topic;
-import pl.allegro.tech.hermes.api.Query;
+import pl.allegro.tech.hermes.api.*;
```

Comment:
IMO these wildcards are too greedy :) please tune your IDE, so explicit imports will be left instead.